public/stylesheet/styles.css

```diff
@@ -0,0 +1,419 @@
+* {
+    box-sizing: border-box;
+}
+
+html{
+   font-size: 16px;
+   height: 100%;
+   margin: 0.5rem;
+}
+
+body{
+  min-height: 99%;
+  position: relative;
+}
+
+body:after{
+  display: block;
+  height: 90%;
+}
+
+/**  HEADER ************************************************************/
+
+header{
+  position: relative;
```

Comment:
If you put properties in alphabetical order, it'll be easy to scan the list for rule you want to change